MAUI/Cartesian-Charts/Trackball.md

```diff
@@ -0,0 +1,299 @@
+
+SfCartesianChart chart = new SfCartesianChart();
+. . .
+ChartLabelStyle labelStyle = new ChartLabelStyle();
+labelStyle.Background = Colors.LightBlue;
+labelStyle.FontSize = 15;
+labelStyle.CornerRadius = 5;
+labelStyle.StrokeWidth = 2;
+labelStyle.Stroke = Colors.Green;
+
+this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Trackball Line Customization
+
+ The [LineStyle]() property provides to customize the trackball line. These options are:
+
+* `StrokeWidth`, of type `double`, used to change the stroke width of the line.
+* `Stroke`, of type `Brush`, used to change the stroke color of the line.
+* `StrokeDashArray`, of type `DoubleCollection`, specifies the dashes to be applied on the line.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+   ...
+    <chart:ChartTrackballBehavior.LineStyle>
+        <chart:ChartLineStyle Stroke="yellow"   
+                              StrokeDashArray="4"/>     
+    </chart:ChartTrackballBehavior.LineStyle>
+...
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+. . .
+ChartLineStyle lineStyle = new ChartLineStyle();
+lineStyle.Stroke = Colors.Yellow;
+lineStyle.StrokeDashArray = 4;
+
+this.Content = chart;
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+### Trackball Markers Customization
+
+The [MarkerSettings]() property provides to customize the trackball markers.The following properties are used to customize the trackball marker.
+
+* `Type`, of type `ShapeType`, used to set the marker shape type.
+* `Stroke`, of type `Brush`, used to change the marker border color.
+* `Fill`, of type `Brush`, used to change the marker background color.
+* `StrokeWidth`, of type `double`, used to change the width of the marker border.
+* `Width`, of type `double`, used to change the width of the marker.
+* `Height`, of type `double`, used to change the height of the marker.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+   ...
+    <chart:ChartMarkerSettings Type="InvertedTriangle"  
+                               Fill="Blue" 
+                               Stroke="Red" 
+                               StrokeWidth="1.5"
+                               Width="15" 
+                               Height="15"/>
+
+...
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+. . .
+ChartMarkerSettings markerStyle = new ChartMarkerSettings()
+{
+   markerStyle.Type = ShapeType.InvertedTriangle,
+   markerStyle.Fill = Colors.Blue,
+   markerStyle.Stroke = Colors.Red,
+   markerStyle.StrokeWidth = 1.5,
+   markerStyle.Width = 15,
+   markerStyle.Height = 15,
+};
+
+this.Content = chart;
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+## Show or Hide the Series Label
+
+This feature is used to show or hide the trackball label of the series by using the [CartesianSeries.ShowTrackballLabel]() property. The default value is True.
```

Comment:
The ShowTrackballLabel property can be used to hide the particular series trackball label. Its default value is true.

Reply:
Changed the content